docs/en/cli.md

```diff
@@ -0,0 +1,475 @@
+# OpenMLDB operation CLI
+
+* [ns client](#ns-client)
+* [tablet client](#tablet-client)
+
+### NS Client
+
+Connecting to the ns client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk, role is the role to be started and needs to be specified as ns_client
+
+```bash
+$ ./bin/openmldb --zk_cluster=172.27.2.52:12200 --zk_root_path=/onebox --role=ns_client
+```
+
+#### use
+
+The `use` command can switch to a database
+
+```
+> use demodb
+```
+
+### showtable
+
+View all tables or specify a table
+
+Command format: `showtable [table_name]`
+
+```
+172.24.4.55:6531 demo_db > showtable
+   name tid pid endpoint role is_alive offset record_cnt memused diskused
+-------------------------------------------------- -------------------------------------------------- --
+  aaa 22 0 172.24.4.55:9971 leader yes 0 0 0.000 8.646 K
+  auto_VCeOIIKA 25 0 172.24.4.55:9971 leader yes 8 4 498.000 9.128 K
+  t1 21 0 172.24.4.55:9971 leader yes 1 1 301.000 9.353 K
+172.24.4.55:6531 demo_db> showtable auto_VCeOIIKA
+   name tid pid endpoint role is_alive offset record_cnt memused diskused
+-------------------------------------------------- -------------------------------------------------- --
+  auto_VCeOIIKA 25 0 172.24.4.55:9971 leader yes 8 4 498.000 9.128 K
+```
+
+#### showtablet
+
+View tablet information (if the serverName and automatic local ip function are used, the endpoint is serverName, and the real_endpoint is "-")
```

Comment:
(if the serverName and automatic local ip function are used, the endpoint is serverName, and the real_endpoint is "-")
=>
(if the serverName is specified and the local IP auto-configuration is enabled, then the endpoint is shown as the serverName and the real_endpoint is shown as "-")

Reply:
Ok, thank you.